task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -1467,6 +1460,28 @@ def main():
             with suppress(Exception):
                 SUPERVISOR_COMMS.socket.close()
 
+def reinit_supervisor_comms() -> StartupDetails:
```

Comment:
```suggestion
def reinit_supervisor_comms() -> None:
```

We do not return do we?

Reply:
Correct, yeah. (I did in an earlier version, but forgot to update this sig)